src/modules/Routing.rb

```diff
@@ -205,6 +205,35 @@ def WriteIPForwarding
       write_ipv6_forwarding(@Forward_v6)
     end
 
+    # Converts routes config as read from system into well-defined format
+    #
+    # Expects list of hashes as param. Hash should contain keys "destination",
+    # "gateway", "netmask", "device", "extrapara"
+    #
+    # Currently it converts "destination" in CIDR format (<ip>/<prefix_len>)
+    # and keeps just <ip> part in "destination" and puts "<prefix_len>" into
+    # "netmask"
+    #
+    # @param <List> list of hashes which keeps routes configuration
+    #
+    # @return list of converted hashes
+    def normalize_routes(routes)
+      return routes if routes.nil? || routes.empty?
+
+      routes.map do |route|
+        subnet, prefix = route["destination"].split("/")
+
+        if !prefix.nil?
+          route["destination"] = subnet
+          route["netmask"] = "/#{prefix}"
+        end
+
+        route
+      end
+
+      routes
```

Comment:
Very nasty sideeffect as it looks like it do nothing as you do not assign map result to routes variable or use `map!`. But in fact change hash in map, so even each do that stuff. Also it is not clear that it modify parameter itself.